### DISCUSSION TITLE
Disclosure in React Aria Components

### DISCUSSION THREAD

**Discussion by p1pah** [adobe/react-spectrum#5788] [Q&A] [answered]
Good morning, 

I am curious if there is plans to release a disclosure component in RAC? If not, what would best practice to replicate something like this...

![image](https://github.com/adobe/react-spectrum/assets/17172000/d79fa94c-454b-48e2-8a8b-d4c6a183c1cf)

**Comment by reidbarber** [comment, marked as answer]
There is an existing issue at https://github.com/adobe/react-spectrum/issues/3817 discussing the disclosure pattern. This design looks similar to the Accordion, which is on the roadmap for us: https://github.com/adobe/react-spectrum/issues/1725. We've also recommended people use the [details](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) HTML element in the past.

**Reply by p1pah** [reply]
thanks @reidbarber appreciate the responsiveness!
